README.md

```diff
@@ -19,7 +19,7 @@ See [here](https://www.lucidchart.com/documents/view/53cd191b-3ca5-4b23-832d-28a
 ## Dependencies
 Before you install DAHLIA, your system should have the following:
 
-- [Ruby](https://www.ruby-lang.org/en/documentation/installation/) 2.2.3
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/) 2.2.3 (Use [RVM](https://rvm.io/rvm/install) or [rbenv](https://github.com/rbenv/rbenv))
 - [Bundler](https://github.com/bundler/bundler) `gem install bundler -v 1.12.5`
```

Comment:
I got a warning with this version saying "Warning: the running version of Bundler is older than the version that created the lockfile. We suggest you upgrade to the latest version of Bundler by running `gem install bundler`."

Is it OK to update it to just `gem install bundler`?

Reply:
Yep go for it!